server/web/api/register.py

```diff
@@ -0,0 +1,112 @@
+from functools import wraps
```

Comment:
i see a lot of fixme and removes and todos in this file. Please mark this file as deprecated or temporary. Further the next step should be to relocate all the methods to their semantic module. For example the login and admin methods relate to "administration" and start scenario to the web module of execution. It is up to you, if you adress the topic in the next implementation or this pr.